anta/models.py

```diff
@@ -244,6 +245,7 @@ class Input(BaseModel):
         """
 
         model_config = ConfigDict(extra="forbid")
+        tags: Optional[List[str]] = ["all"]
```

Comment:
wondering if we should not prefix this in case someone would need `tags` in a test one day?